src/view/gui/fMainWindow.cpp

```diff
@@ -4523,20 +4523,36 @@ void fMainWindow::CallGeneratePopualtionAtlas(const std::string inputFileName, c
   } 
   //put the data in respective vectors
   std::vector< std::string > patient_ids, image_paths, atlas_labels;
+  std::vector<int> image_available;
   for (int j = 1; j < allRows.size(); j++)
   {
+    int patient_id_index = -1;
+    int images_index = -1;
+    int atlas_labels_index = -1;
     for (size_t k = 0; k < allRows[0].size(); k++)
     {
       auto check_wrap = allRows[0][k];
       std::transform(check_wrap.begin(), check_wrap.end(), check_wrap.begin(), ::tolower);
-
       if (check_wrap == "patient_ids")
-        patient_ids.push_back(allRows[j][k]);
+        patient_id_index = k;
```

Comment:
Trying to understand this change: this seems to indicate that ```patient_id_index``` gets overwritten, is that right?

Reply:
yes. this gets over-written.